packages/flame/test/components/component_test.dart

```diff
@@ -1474,18 +1473,19 @@ void main() {
         final component0 = Component();
         expect(component0.children.strictMode, false);
 
-        Component.childrenFactory = () => QueryableOrderedSet(
-              OrderedSet.comparing(Comparing.on((e) => e.priority)),
-              strictMode: false,
+        Component.childrenFactory = () => OrderedSet.mapping<num, Component>(
+              (e) => e.priority,
+              // ignore: avoid_redundant_argument_values
+              strictMode: true,
```

Comment:
Can we just remove?

Reply:
Then it's not as easy to read and see what is happening in the test, so I choose to have it there explicitly.